### DISCUSSION TITLE
Remnant Wormhole Failure

### DISCUSSION THREAD

**Discussion by SultanRTG** [endless-sky/endless-sky#8313] [Q&A] [answered]
Recently I noticed that the Edusa-Pereganor wormhole was not a location that I could land on/use. Can someone provide some insight as to why?

**Comment by Zitchas** [comment, marked as answer]
Which version of the game are you playing?
Are you using any plug-ins?

**Reply by SultanRTG** [reply]
No, I just realized that I wasn't using a Keystone.

**Reply by Zitchas** [reply]
Ah, that would do it. Glad you figured it out.
